lib/Bio/KBase/ObjectAPI/PATRICStore.pm

```diff
@@ -688,7 +688,8 @@ sub save_model {
     }
     $summary->{status} = "complete";
     $summary->{status_timestamp} = Bio::KBase::utilities::timestamp();
-	$self->helper()->update_model_meta($ref,$summary);
+	#$self->helper()->update_model_meta($ref,$summary);
+	Bio::ModelSEED::patricenv::call_ws("update_metadata",{objects => [[$ref,$summary]]});
```

Comment:
Why not fix the helper function?

Reply:
True.

Reply:
Actually i changed my mind again, I remembered, patricenv is consistently called throughout PATRICStore.pm so I'd like to keep it this way.